packages/airnode-protocol/contracts/rrp/WithdrawalUtils.sol

```diff
@@ -28,6 +28,8 @@ contract WithdrawalUtils is IWithdrawalUtils {
         external
         override
     {
+        require(airnode != address(0), "Airnode address zero");
```

Comment:
If a withdrawal succeeds, the source wallet will be close to empty (EIP-1559) or completely empty (legacy gas pricing), in which case future attempts to withdraw will fail as Airnode checks the balance before attempting to process a withdrawal.

Reply:
`require` gas cost is typically not a consideration. The main reason I'm not doing the same check in `fulfillWithdrawal()` is that https://github.com/api3dao/airnode/blob/322e91b620da13fcd02b2667875c333a3c8e0855/packages/airnode-protocol/contracts/rrp/WithdrawalUtils.sol#L65-L69 already does it implicitly, i.e., it is guaranteed that this will revert if `airnode` is `address(0)`

Reply:
Documented here https://github.com/api3dao/airnode/pull/748/commits/39014e3be1289533d58a57ead51a41ec73d76df2

Reply:
ah yeah, that makes sense